modes/unimpaired/evil-collection-unimpaired.el

```diff
@@ -78,12 +78,13 @@
 (defun evil-collection-unimpaired-insert-space-above (count)
   "Insert space above current line"
   (interactive "p")
-  (dotimes (_ count) (save-excursion (evil-insert-newline-above))))
+  (save-excursion (dotimes (_ count) (evil-insert-newline-above)))
+  (if (bolp) (forward-char count)))
```

Comment:
Prefer `when` to `if` when there is no "else" form.  Otherwise looks good to me!